source/slang/slang-lower-to-ir.cpp

```diff
@@ -2947,6 +2947,16 @@ struct ExprLoweringVisitorBase : ExprVisitor<Derived, LoweredValInfo>
         return info;
     }
 
+    LoweredValInfo visitJVPDerivativeOfExpr(JVPDerivativeOfExpr* expr)
+    {
+        // Dummy expr.. just emit the decl-ref of the inner function.
+        LoweredValInfo info = emitDeclRef(
```

Comment:
I assume you will modify this later to generate an `IRJVPDerivative` inst that has the result of `emitDeclRef` as its operand and return that inst here.